data/ecosystem/distributions.yaml

```diff
@@ -0,0 +1,36 @@
+# cSpell:ignore Splunk AWS ADOT Grafana observIQ BindPlane RHOSDT RedHat Sumo Liatrio
+- name: Splunk Distribution of OpenTelemetry Collector
+  url: https://github.com/signalfx/splunk-otel-collector
+  docsUrl: https://docs.splunk.com/observability/en/gdi/opentelemetry/opentelemetry.html
+  components: [Collector]
+  oss: true
+- name: AWS Distribution for OpenTelemetry Collector (ADOT Collector)
+  url: https://github.com/aws-observability/aws-otel-collector
+  docsUrl: https://aws-otel.github.io/docs/getting-started/collector
+  components: [Collector]
+  oss: true
```

Comment:
I wonder if we would even accept non-OSS distributions, what would be the purpose of that?

Reply:
Hmmm, good point. Let me remove that field.